### DISCUSSION TITLE
Deploy Dicom-Server on privacy environment on Azure

### DISCUSSION THREAD

**Discussion by japineda3** [microsoft/dicom-server#733] [Q&A]
Hi,
    I am trying to deploy tis service on Azure with maximum privacy but I don´t know if it is possible to deploy the Dicom-server without use the arm template , because I would like to deploy the resources on Azure with privacy. For example, I would like to configure the security on SQL Azure to allow only connections by private endpoint not public endpoint as now . The same with the rest of resources.   I don´t find where you put the database string connection on the dicom web service to change this. Please , anyone could help me with this? thanks.

**Comment by jovinson-ms** [comment]
Sorry for the late reply - we are more likely to catch questions like this if they're created as an Issue. 

You can set the SQL connection string in the `appsettings.json` `SqlServer` [configuration section](https://github.com/microsoft/dicom-server/blob/746e6e9e099b3436d1609219259dbe6ca9a2510f/src/Microsoft.Health.Dicom.Web/appsettings.json#L98), like it is set in [development app settings](https://github.com/microsoft/dicom-server/blob/746e6e9e099b3436d1609219259dbe6ca9a2510f/src/Microsoft.Health.Dicom.Web/appsettings.Development.json#L17). 

In production, you can set this value via an App Service app setting, environment variable, or other configuration options.
